packages/teams-js/src/public/sharing.ts

```diff
@@ -243,10 +243,6 @@ export namespace sharing {
         getApiVersionTag(sharingTelemetryVersionNumber_v2, ApiName.Sharing_History_GetContent),
         'sharing.history.getContent',
       );
-      contentDetails.map((contentDetails) => {
-        validateUuid(contentDetails.author);
-        validateUuid(contentDetails.threadId);
```

Comment:
>threadId [](http://example.com/codeflow?start=36&length=8)

Are there any restrictions on these IDs at all? Or can they literally be any string?

Reply:
Yes, they are treated as any strings in all parts of the Teams code. The different formats for userIds and threadIds are used depending on the scenario, such as enterprise, life, or government clouds.